README.md

```diff
@@ -24,6 +24,19 @@ Dharma Seed is dedicated to preserving and sharing the spoken teachings of Thera
 <img src="doc/images/device-2016-05-19-083004.png" width="160">
 <img src="doc/images/device-2016-05-19-083049.png" width="160">
 
+## Contributing
+
+Fork the repo using Github and clone your fork locally. To run the project, plug in your device and click the green 'Play' button in
+Android Studio. Once you're ready to start making changes, create a branch: `git ch -b <branch_name>`. Name it something related to the
+changes you plan on working on, like `update-db`. Once you've made your changes, run the tests to make sure you didn't break
+anything and then add and commit to git. Once you've made as many commits as you need, push them to Github and create a PR.
+
+### Running the Tests
+
+As of right now, all of our tests are Instrumentation Tests (as opposed to Unit Tests) which means that they require a device to run.
+They are located in `app/src/androidTest/`. Make sure your device is plugged in (you can also use an emulator but it's _much_ slower), right click the package `org.dharmaseed.android (androidTest)` in the project window of Android Studio, and select `Run tests in org.dharmaseed.android (androidTest)`.
```

Comment:
Just curious, how slow is it for you running the emulator?  The tests finish in less than a second for me.  I'm running the emulator on linux through kvm (hardware virtualization acceleration) though, so maybe that's the difference.

Reply:
I guess I just meant start-up time, since firing up the emulator takes several minutes but running them is quick. I'll remove that part, since it doesn't add anything anyway.

Reply:
removed

Reply:
:+1: